libsolidity/interface/FileReader.cpp

```diff
@@ -31,7 +31,6 @@
 #include <functional>
 
 using solidity::frontend::ReadCallback;
-using solidity::langutil::InternalCompilerError;
```

Comment:
Can you now get rid of `InteralCompilerError` from `Exceptions.h`?

Reply:
What do you mean? We're still using it, just via `solAssert()` rather than throwing it directly.